src/client/docs/import.md

```diff
@@ -0,0 +1,100 @@
+# Bohrdaten importieren
+
+Mit der Import-Funktion können geologische Bohrdaten mit einer zuvor erstellten CSV-Datei in einem Rutsch importiert werden. 
+
+## Anleitung
+
+### Schritt 1: CSV-Datei vorbereiten
+
+Zunächst sollte die CSV-Datei den Anforderungen und dem Format entsprechen, wie im Abschnitt [Format und Anforderungen an die CSV-Datei](#format-und-anforderungen-an-die-csv-datei) beschrieben.
+
+### Schritt 2: Navigieren zum Import-Bereich
+
+1. In der Webapplikation anmelden.
+2. In die Editor-Ansicht wechseln.
+3. Unten links auf die Schaltfläche _Importieren_ klicken.
+
+### Schritt 3: CSV-Datei hochladen
+
+1. Schaltfläche _Datei auswählen_ anklicken und die vorbereitete CSV-Datei auswählen.
+2. Unter _Arbeitsgruppe_ die Arbeitsgruppe auswählen, in welche die Bohrdaten importiert werden sollen.
+3. Import-Prozess mit einem Klick auf _Importieren_ starten.
+4. Warten, bis der Upload abgeschlossen ist und die Daten in der Anwendung verfügbar sind.
+
+## Format und Anforderungen an die CSV-Datei
+
+Die CSV-Datei muss den folgenden Anforderungen und dem Format entsprechen, damit sie erfolgreich in die Webapplikation importiert werden kann:
+
+* Die Datei muss im CSV-Format vorliegen und als Trennzeichen wird der Strichpunkt (;) verwendet.
+* Die Datei muss im UTF-8-Format gespeichert sein.
+* Die erste Zeile der CSV-Datei muss die Spaltenüberschriften enthalten.
+* Die Spaltenüberschriften müssen den vorgegebenen Feldnamen aus der Webapplikation entsprechen.
```

Comment:
PP/Q:

Ich würde hier den Verweis auf die Feldnamen _**in der Webapplikation**_ weglassen. Da ein 'normaler' Benutzer nur die Beschreibung des Felds sieht, was ja dann aber falsch wäre (`purpose_id | Bohrzweck`). Besser auf die Tabelle unten verweisen?